EducationalAssignments/UndoPartTwo.md

```diff
@@ -119,6 +119,14 @@ write the security layer, then you solve the puzzle - try to bypass it.  If your
 puzzle is "good enough", no one will be able to break it, no matter what.  
 
 
+> **Note**: When creating tests, it's best practice to separate them into
+> different files. If there is a flaw in any part of a test, it can invalidate
```

Comment:
Edit this to be more precise.  

"it can invalidate the entire thing." -> "we will discard the entire test file."  

The sentences following also need to be more precise to cover this.

Reply:
> Note: When creating tests, the best practice is to separate them into
> different files. Each test file should focus on testing one specific scenario
> or functionality. If there is a flaw in any part of a test file, we will
> discard the entire test file. This modular approach helps isolate issues - if
> there is a flaw in one test file, it will not affect or invalidate the other
> test files, since each file is independent and testing distinct functionality.

Prof @JustinCappos, WDYT?